apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -174,7 +174,22 @@ export const KanbanView = ({ kanbanBoardTasks, isLoading }: { kanbanBoardTasks:
 		};
 	}, []);
 
-	if (!enabled) return null;
+	// const [editStatus, setEditStatus] = useState(); // used for status
```

Comment:
@Anishali2 can we remove that commented-out code? Or do you want to work on it in another PR?

Reply:
Sir I can remove that in next PR